app/layout.tsx

```diff
@@ -46,7 +46,7 @@ export default function RootLayout({
         }}
       >
         <AppRouterCacheProvider>
-          <ThemeProvider theme={theme} defaultMode="dark">
+<ThemeProvider theme={theme}>
```

Comment:
```suggestion
          <ThemeProvider theme={theme} defaultMode="dark">
```
I made the stylistic choice of making dark mode the default theme :-)